test/integration/badgeController.test.js

```diff
@@ -0,0 +1,26 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
+const app = require('../../server')
+
+chai.use(chaiHttp)
+
+describe('Badges', function () {
+  describe('GET /badges', function () {
+    it('Should get all the list of badges', function (done) {
+      chai
+        .request(app)
+        .get('/badges')
+        .end((err, res) => {
+          if (err) { return done() }
+          expect(res).to.have.status(200)
+          expect(res.body).to.be.a('object')
+          expect(res.body.message).to.equal('Badges returned successfully!')
+          expect(res.body.badges).to.be.a('array')
+
+          return done()
+        })
+    })
+  })
+})
```

Comment:
Where are we adding the badges in the database to `GET` them? 🤔
How will this test case pass on CI? 🤔

Reply:
Ankush is going to add badges manually in database until we meet the authorization in place.

Reply:
> 
> 
> Ankush is going to add badges manually in database until we get the authorization in place.

I guess the CI also uses firebase emulators that's why we install it on every commit.
I meant the CI doesn't use the production db, right?
Feel free to correct me 😄

Reply:
Yes, seems like CI uses emulator, check test.yml file in .git/workflow
CI fetching and adding data in prod DB won't be a good idea

Reply:
That's what I'm saying 😅 
The prod DB will have the badges details
but the firebase emulator wont and hence the test wouldn't pass 😅 

Let me know if I'm wrong 😄

Reply:
Oh, that's the convention, even if the GET Endpoint returns an empty array, the response code will still be 200, and the request is considered fulfilled.

Reply:
Cool